requirements-dev.txt

```diff
@@ -3,17 +3,18 @@ behave==1.2.6
 behaving==2.0.0
 Appium-Python-Client<=0.52
 ckanapi==4.3
+ckantoolkit>=0.0.4
 factory-boy==2.12.0
 flake8==3.8.3
 mock
 parameterized==0.8.1
 pytest-ckan
-splinter>=0.13.0
+splinter>=0.13.0,<0.17
```

Comment:
would we need to prepare for py2, py3 where splinter 0.17+ is available?

Reply:
If we allow 0.17+, then Py2 builds will break. We can only lift that restriction once py2 is discarded.